src/chi-vue/src/components/column-customization/ColumnCustomizationModalContent.tsx

```diff
@@ -172,6 +173,30 @@ export default class ColumnCustomizationContent extends Vue {
     }
   }
 
+  _handleSelectColumn() {
+    const columnsSelected: string[] = [];
+    const filterSelectedColumns: DataTableColumn[] = [];
+
+    if (this._selectedStandardColumns) {
+      Array.from((this._selectedColumnsComponent?.$refs.select as HTMLSelectElement).selectedOptions).forEach(
+        (option: HTMLOptionElement) => {
+          columnsSelected.push(option.value);
+        }
+      );
+
+      columnsSelected.forEach((columnSelected: string) => {
+        this._selectedStandardColumns?.forEach((standardColumn: DataTableColumn) => {
+          if (columnSelected === standardColumn.name) {
+            filterSelectedColumns.push(standardColumn);
+          }
+        });
+      });
+    }
+
+    this._isSelectedColumnLocked = filterSelectedColumns.some((column: DataTableColumn) => column.locked);
```

Comment:
It seems no locked columns can exist in the filterSelectedColumns. It comes from columnsSelected, which comes from _selectedStandardColumns that does not have locked columns.